<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Video</class>
 <widget class="QDialog" name="Video">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widgetTop" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>45</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>45</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="btnReturn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labTitle">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnMsg">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>91</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="set_ch1">
    <item>
     <widget class="QComboBox" name="record_enable_ch1">
      <item>
       <property name="text">
        <string>开</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>关</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="fps_ch1">
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>23</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="resolution_ch1">
      <item>
       <property name="text">
        <string>QCIF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CIF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WCIF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>D1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WD1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>720P</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1080P</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="quality_ch1">
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="osd_enable_ch1">
      <item>
       <property name="text">
        <string>开</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>关</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Ch1_name"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>121</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="set_name">
    <item>
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string>视频通道</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>录像使能</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>录像帧率</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>录像分辨率</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>录像清晰度</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>录像OSD使能</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>通道名称</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>501</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="channel_name">
    <item>
     <widget class="QLabel" name="label_ch1">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>CH1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_ch2">
      <property name="text">
       <string>CH2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_ch3">
      <property name="text">
       <string>CH3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_ch4">
      <property name="text">
       <string>CH4</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>91</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="set_ch2">
    <item>
     <widget class="QComboBox" name="record_enable_ch2">
      <item>
       <property name="text">
        <string>开</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>关</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="fps_ch2">
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>23</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="resolution_ch2">
      <item>
       <property name="text">
        <string>QCIF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CIF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WCIF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>D1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WD1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>720P</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1080P</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="quality_ch2">
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="osd_enable_ch2">
      <item>
       <property name="text">
        <string>开</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>关</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Ch2_name"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>140</y>
     <width>91</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="set_ch3">
    <item>
     <widget class="QComboBox" name="record_enable_ch3">
      <item>
       <property name="text">
        <string>开</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>关</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="fps_ch3">
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>23</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="resolution_ch3">
      <item>
       <property name="text">
        <string>QCIF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CIF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WCIF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>D1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WD1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>720P</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1080P</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="quality_ch3">
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="osd_enable_ch3">
      <item>
       <property name="text">
        <string>开</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>关</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Ch3_name"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>140</y>
     <width>91</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="set_ch4">
    <item>
     <widget class="QComboBox" name="record_enable_ch4">
      <item>
       <property name="text">
        <string>开</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>关</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="fps_ch4">
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>23</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="resolution_ch4">
      <item>
       <property name="text">
        <string>QCIF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CIF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WCIF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>D1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WD1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>720P</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1080P</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="quality_ch4">
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="osd_enable_ch4">
      <item>
       <property name="text">
        <string>开</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>关</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Ch4_name"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ParaSet">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>440</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>参数设置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
